Cargo.toml

```
@@ -13,20 +13,20 @@ readme = "README.md"
 license = "MIT"
 
 [dev-dependencies]
-nix = "~0.11"
-tempdir = "~0.3"
+nix = "^0.23"
```

Caret requirement is the [default requirement in cargo](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio). So we may make it simpler 
```suggestion
nix = "0.23"
```

---

Yes, we can remove that.
@cholcombe973 I corrected it to format `"x.y.z"` instead of `^x.y.z`